Patch by Robert Scheck <redhat@linuxnetz.de> for tar >= 1.15.1, which makes tar
rebuildable using gcc 4.

--- tar-1.15.1/tests/genfile.c			2004-09-08 13:50:20.000000000 +0200
+++ tar-1.15.1/tests/genfile.c.gcc4		2005-03-13 16:25:15.000000000 +0100
@@ -60,8 +60,8 @@
 /* Block buffer for sparse file */
 char *buffer;
 
-static const char *argp_program_version = "genfile (" PACKAGE ") " VERSION;
-static const char *argp_program_bug_address = "<" PACKAGE_BUGREPORT ">";
+const char *argp_program_version = "genfile (" PACKAGE ") " VERSION;
+const char *argp_program_bug_address = "<" PACKAGE_BUGREPORT ">";
 static char doc[] = N_("genfile generates data files for GNU paxutils test suite");
 
 static struct argp_option options[] = {
